ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -101,7 +112,6 @@ public void SetResizeMode(Border view, string resizeMode)
         public void SetSource(Border view, string source)
```

Comment:
We'll have to move this to a shadow node once we start adding batches of properties (e.g., tint, background color, etc.)